### ISSUE TITLE
Various Fingerprint HAL issues

### ISSUE THREAD

**Issue by regalstreak** [CyanogenMod/android_device_oneplus_oneplus2#3]
http://pastebin.com/VH0TyWik
for cm13.0 branch.

Ill build without fingerprint HAL and report if any other errors :)
For that time lets disable it.

**Comment by dansou901** [comment]
In your paste I see fp hal for 1020, but Oneplus2 is using 1015 if I'm not mistaken...

**Comment by manups4e** [comment]
the fingerprint sensors of oneplus2 is 1150

**Comment by regalstreak** [comment]
But there are errors while compiling. The fingerprint hal is forked from
mokee. We need to get perfect sources for the hal for cm13

**Comment by manups4e** [comment]
cm13 as android 6.0 use opensource fingerprint hal that means that it will be supported natively without the need of this hal (that is not even our hal)

**Comment by regalstreak** [comment]
So did you get a perfectly booting cm13 till now? Mine is stuck at bootloop.

**Comment by manups4e** [comment]
almost.. my problem is that i have fatal error on pm-proxy binary and art doesn't dex2oat jar files.. for the rest everything seems to be ok.. but those 2 prevents everything from booting

**Comment by regalstreak** [comment]
Same. pm-proxy error :(

**Comment by manups4e** [comment]
that's why i keep telling that using oxygenOS 5.1.1 sources will never work.. we need to wait for Marshmallow blobs to get cm13 to boot. using sources from other devices is not working (and i don't think it's a good idea)

**Comment by regalstreak** [comment]
Yes many blobs are wrong for cm13 if uve forked from themuppets git as they
wont work.

**Comment by manups4e** [comment]
no i'm using grarak ones..

**Comment by Seraph08** [comment]
in your guy's boot loops did you ever get past the oneplus screen?

**Comment by manups4e** [comment]
Yeah i have.. The problem is after the bootlogo
Il 06/dic/2015 10:51 AM, "David Vandenburgh" <notifications@github.com> ha
scritto:

**Comment by Seraph08** [comment]
I'll be spending part of today playing catch up then.

**Comment by regalstreak** [comment]
@Seraph08 you building aospa?

**Comment by Seraph08** [comment]
yeah, been working on it

**Comment by Seraph08** [comment]
@regalstreak hows that booting

**Comment by regalstreak** [comment]
i have made cm13 to boot! Oh wow so happy. Bump in the xda thread here:
http://forum.xda-developers.com/oneplus-2/general/cyanogenmod-13-android-marshmallow-t3256275

**Comment by manups4e** [comment]
Good news.. Now the easy part was done :) the bugfixing begins
Il 08/dic/2015 01:02 PM, "Neil" <notifications@github.com> ha scritto:

**Comment by regalstreak** [comment]
yes! :D Im uploading vendor tree. WIll commit stuff. Current one is a dirty build. Had to delete some files and edit /system/vendor stuff.

**Comment by manups4e** [comment]
This is not good we need to boot it without Frankenstein builds
Il 08/dic/2015 01:15 PM, "Neil" <notifications@github.com> ha scritto:

**Comment by regalstreak** [comment]
I am doing the work now. Uploading vendor sources gonna take a year. Speed: 20kBps :-1: :3
Have got it covered. Dont worry!

**Comment by Seraph08** [comment]
I'm a patient kind a guy.

**Comment by hughesjs** [comment]
Just like to say that I've got it registering when a finger is on the sensor (doesn't register a knuckle as a finger) and can register fingers in settings. However, it is always showing up as unrecognized whilst trying to unlock the device. Have any of you got any idea of what the cause might be? If not, I'll go digging and see what I can do.
